packages/reva-api/domain/features/getCandidateByEmail.ts

```diff
@@ -0,0 +1,17 @@
+import { Candidate } from "@prisma/client";
```

Comment:
On en a déjà parlé mais du coup ca crée un lien fort entre le "domain" et la partie tech.
On peut laisser pour le moment tant qu'on reflechi a ce qu'on va faire in fine

Reply:
👍